lib/foreman_bootdisk/engine.rb

```diff
@@ -31,12 +31,10 @@ class Engine < ::Rails::Engine
         requires_foreman '>= 1.9'
 
         security_block :bootdisk do |map|
-          permission :download_bootdisk, {:'foreman_bootdisk/disks' => [:generic, :host, :full_host, :help],
+          permission :download_bootdisk, {:'foreman_bootdisk/disks' => [:generic, :host, :full_host, :subnet, :help],
                                           :'foreman_bootdisk/api/v2/disks' => [:generic, :host]}
         end
 
-        role "Boot disk access", [:download_bootdisk]
```

Comment:
Can you restore this please?  I don't know why you were putting view_hosts and view_subnets permissions in it anyway, so I don't see the need to use a DB seed.